compute/elastic-metal/how-to/use-commitment-fee-payback.mdx

```diff
@@ -0,0 +1,48 @@
+---
+meta:
+  title: How to use the commitment fee payback
+  description: This page explains how to to use the commitment fee payback for Elastic Metal servers
+content:
+  h1: How to use the commitment fee payback
+  paragraph: This page explains how to use the commitment fee payback for Elastic Metal server servers
+tags: elastic-metal payback commitment fee 
+dates:
+  validation: 2022-12-01
+  posted: 2022-12-01
+categories:
+  - compute
+---
+
+
+You pay a commitment fee when you order your Elastic Metal server with monthly billing. This fee covers the cost of the setup of your machine, and you profit from a lower monthly price. You are entitled to a commitment fee payback (credit) if your server is up and running for 12 consecutive months without termination.
+
+The amount of the credit is the price of the commitment fees at the time of purchase. It can be used for all Scaleway Cloud products (except Elastic Metal servers and their associated options) and is valid for six months. You receive the credit once per physical server, and it is automatically credited to your Scaleway customer account. 
```

Comment:
```suggestion
The amount of the credit is the price of the commitment fees at the time of purchase. It can be used for all Scaleway Cloud products (except Elastic Metal servers and their associated options) and is valid for six months. You receive the credit once per physical server, and it is automatically credited to your Scaleway account. 
```